include/sound/compress_driver.h

```diff
@@ -34,11 +34,14 @@ struct snd_compr_ops;
  * @total_bytes_transferred: cumulative bytes transferred by offload DSP
  * @sleep: poll sleep
  * @private_data: driver private data pointer
+ * @dma_area: DMA area
```

Comment:
definition?

Reply:
Same here, can add something else, but this is based on pcm.h.

Reply:
Ack.

Reply:
Fixed in v5.